chart/values.yaml

```diff
@@ -0,0 +1,102 @@
+role: for-test
+replicas: 1
+
+images:
+  etcd: quay.io/coreos/etcd:v3.3.9
+  etcd-backup-restore: eu.gcr.io/gardener-project/gardener/etcdbrctl:0.3.0
+
+backup:
+  schedule: "*/5 * * * *" # cron standard schedule
+  maxBackups: 7 # Maximum number of backups to keep (may change in future)
+  storageProvider: "Local"  # Abs,Gcs,S3,Swift empty means no backup,
+  backupSecret: etcd-backup
+  storageContainer: ""
+  env: []         # Follow comments below
+  volumeMounts: []
+
+# TLS secrets must be created before running this Chart. Each secret should
+# have below data fields:
+# * tlsCaSecretName should have "ca.crt"
+# * tlsServerSecretName should have "tls.crt" and "tls.key",
+#                       CN should be "etcd-<role>", and
+#                       hosts should include "etcd-<role>-0"
+# * tlsClientSecretName should have "tls.crt" and "tls.key"
+tlsCaSecretName: ca-etcd
```

Comment:
How about restructuring it as:
```yaml
tls:
 caSecret: ca-etcd
 serverSecret: etcd-server-tls
 clientSecret: etcd-client-tls
```

Also, internally, in template files, have if-check `{{ if .Value.tls }}` then only embed tls related code. Otherwise go without SSL.